### ISSUE TITLE
Improve keyboard navigation with KP_BEGIN and KP_END

### ISSUE THREAD

**Issue by Hartmnt** [mumble-voip/mumble#6479]
### Description

In #5984 the accessibility of the entire application was overhauled. Mumble was made keyboard navigable. However, I failed to notice that ``KP_BEGIN`` and ``KP_END`` were also used by screen reader users for keyboard navigation.

Thanks to user squeaktoy for bringing this to my attention: https://github.com/mumble-voip/mumble/pull/5984#issuecomment-2123446439

We need to go through all UI elements and see how they react to pressing ``KP_BEGIN`` and ``KP_END`` (or "Home" and "End")

### Mumble version

1.5.634

### Mumble component

Client

### Reproducible?

Yes

**Comment by squeaktoy** [comment]
I did notice that `KP_BEGIN` works differently on Windows/NVDA than it does on Linux/Orca. On Orca it reads out useless info while on Windows it does read what you would get if you focus the element through regular navigation.
